drivers/hmis/spec/factories/hmis/workflow_definition/nodes.rb

```diff
@@ -14,7 +14,14 @@
 
   factory :hmis_workflow_definition_end_event, class: 'Hmis::WorkflowDefinition::EndEvent' do
     sequence(:name) { |n| "End Event #{n}" }
-    trigger_config { [] }
+    trigger_config do
```

Comment:
Added a default trigger_config here since end events are not valid without trigger config. https://github.com/greenriver/hmis-warehouse/blob/tt-ce-spike-3/drivers/hmis/app/models/hmis/workflow_definition/end_event.rb#L8